api/api.py

```diff
@@ -35,46 +36,135 @@ def rides_tuple_to_dict(ride_info:tuple) -> dict:
 		'average_power': ride_info[13]
 	}
 
+def users_tuple_to_dict(user_info:tuple) -> dict:
+	return { 
+		'user_id':user_info[0],
+		'first_name':user_info[1],
+		'last_name':user_info[2],
+		'gender':user_info[3],
+		'address':user_info[4],
+		'date_of_birth':user_info[5],
+		'email_address':user_info[6],
+		'height_cm':user_info[7],
+		'weight_kg':user_info[8],
+		'account_create_date':user_info[9],
+		'bike_serial':user_info[10],
+		'original_source':user_info[11]
+	}
+
+
+
 # # GET /ride/:id
 # Get a ride with a specific ID
 @app.route("/ride/<int:ride_id>", methods=["GET"])
 def get_ride_info(ride_id:int):
-	ride_info_tuple = sql.get_list(
-		f"""
-			SELECT *
-			FROM rides
-			WHERE rides_id = {ride_id}
-		""")[0]
-	ride_info = rides_tuple_to_dict(ride_info_tuple)
-	return jsonify(ride_info)
+	try:
+		ride_info_tuple = sql.get_list(
+			f"""
+				SELECT *
+				FROM rides
+				WHERE rides_id = {ride_id}
+			"""
+		)[0]
+		if ride_info_tuple:
+			ride_info = rides_tuple_to_dict(ride_info_tuple)
+			return jsonify(ride_info)
+		else:
+			return jsonify({"output": 'No matching records'})
+	except Exception as e:
+		return jsonify({'error':e})
```

Comment:
I've branched and made the changes. Doing number of seconds since 1st Feb 2023 as the ride_id (any fixed date in the past works really). Number is now in the 100,000s instead of the millions+ it was before!